.circleci/config.yml

```diff
@@ -251,29 +248,4 @@ workflows:
       - forcehybrid-android
       - forcereact-android
       - forceios-sfdx
-      - forcehybrid-ios-sfdx
-      - forcereact-ios-sfdx
       - forcedroid-sfdx
-      - forcehybrid-android-sfdx
-      - forcereact-android-sfdx
-
-  # Build everything iOS wtih Beta Xcode at 9:00 PM PST Wednesday and Sunday
-  weekly-build-all-apps-ios-beta:
```

Comment:
NIT: I changed the triggers in CircleCI so we don't have to remove this but its not a big deal.

Reply:
I guess I can put it back so that we only need to re-enable the trigger when the next Xcode beta comes out.